AI/libs/utils/save_reports_to_db.py

```
@@ -1,53 +1,56 @@
+ASSETS_ROW_ID = os.getenv("ASSETS_ROW_ID", "1")
+
+# ----- 유틸 -----
 def utcnow() -> datetime:
     return datetime.now(timezone.utc)
 
+def _to_decimal(x) -> Decimal:
+    if isinstance(x, Decimal):
+        return x
+    try:
+        return Decimal(str(x))
+    except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)